ming_sim/beat_orchestration.py

```
@@ -735,6 +757,30 @@ def start_close_scene_on_registry(
         scaffold_owned = True
 
     try:
+        # #1585：末位仍在场则先落 TAG_EXIT 垫位，再与 closing 同桶并行生成。
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)